sympy/tensor/array/expressions/conv_array_to_indexed.py

```diff
@@ -1,84 +1,23 @@
-import collections.abc
-import operator
-from itertools import accumulate
+from functools import wraps
 
-from sympy import Mul, Sum, Dummy, Add
-from sympy.tensor.array.expressions import PermuteDims, ArrayAdd, ArrayElementwiseApplyFunc, Reshape
-from sympy.tensor.array.expressions.array_expressions import ArrayTensorProduct, get_rank, ArrayContraction, \
-    ArrayDiagonal, get_shape, _get_array_element_or_slice, _ArrayExpr
-from sympy.tensor.array.expressions.utils import _apply_permutation_to_list
+from sympy.tensor.array.expressions import from_array_to_indexed
+from sympy.utilities.exceptions import sympy_deprecation_warning
 
 
-def convert_array_to_indexed(expr, indices):
-    return _ConvertArrayToIndexed().do_convert(expr, indices)
+def _conv_to_from_warning():
+    sympy_deprecation_warning(
+        "module has been renamed by replacing 'conv_' with 'from_' in its name",
+        deprecated_since_version="1.11",
+        active_deprecations_target="deprecated-conv-array-expr-module-names",
+    )
 
 
-class _ConvertArrayToIndexed:
+def _conv_to_from_decorator(f):
+    @wraps(f)
```

Comment:
There already is a `deprecated` decorator for doing precisely this:
https://docs.sympy.org/dev/guides/contributing/deprecations.html#adding-the-deprecation-to-the-code